pulumi-language-dotnet/main.go

```diff
@@ -819,7 +819,7 @@ func (host *dotnetLanguageHost) constructEnv(req *pulumirpc.RunRequest, config,
 	maybeAppendEnv("stack", req.GetStack())
 	maybeAppendEnv("pwd", req.GetPwd())
 	maybeAppendEnv("dry_run", strconv.FormatBool(req.GetDryRun()))
-	maybeAppendEnv("query_mode", strconv.FormatBool(req.GetQueryMode()))
+	maybeAppendEnv("query_mode", strconv.FormatBool(req.GetQueryMode())) //nolint:staticcheck // Still needed.
```

Comment:
This can just be `maybeAppendEnv("query_mode", "false")` the engine will only ever send false now, but the runtime annoyingly expects this to be set in the environment so we'll have to keep adding it forever.